src/components/pages/CurriculumInfo/tabs/OverviewTab/OverviewTab.tsx

```diff
@@ -9,13 +9,13 @@ import BrushBorders from "@/components/SpriteSheet/BrushSvgs/BrushBorders/BrushB
 import AvatarImage from "@/components/AvatarImage/AvatarImage";
 import Icon from "@/components/Icon/Icon";
 import Typography from "@/components/Typography/Typography";
-import { CurriculumOverviewTabData } from "@/node-lib/curriculum-api-2023";
+import { CurriculumOverviewMVData } from "@/node-lib/curriculum-api-2023";
```

Comment:
Any reason for this naming change? It makes sense but makes things more inconsistent overall (like in the `[tab].tsx` file).

Reply:
I mainly changed this for clarity between the MV and Sanity data - the schemas were confusingly named - but happy to change it back to Tab data. Would be good to think about this kind of situation because I wasn't sure of convention of both sanity and API data in there, and how we might structure it to be more cohesive with the rest of OWA

Reply:
Yeah I see that. We should align on a method and use that consistently for all the API requests. Happy to keep this one as is and we can circle back?